libsql-sqlite3/src/wal.c

```diff
@@ -247,13 +247,28 @@
 ** that correspond to frames greater than the new K value are removed
 ** from the hash table at this point.
 */
-#include "sqliteInt.h"
+#include <sqlite3.h>
```

Comment:
nope - now you risk it will be taken from one of the standard paths like `/usr/include/`, and it's not what you want here. Why is this header needed? If just for a few types => forward-declare them